### DISCUSSION TITLE
Pass-through Events

### DISCUSSION THREAD

**Discussion by 3lrv-ryan** [statelyai/xstate#1678] [Q&A] [answered]
Is it possible to pass-through events from a parent machine to a child machine without having to define each and every event name? I've set up an example [here](https://codesandbox.io/s/ping-pong-v3xls) where the `pongMachine` from the ping-pong example supports a STOP event. The parent `pingMachine` should accept and pass through any events it doesn't know. However, I've no idea whether this is supported (or should be).

**Comment by 3lrv-ryan** [comment, marked as answer]
Looks like I may have guessed the right thing after all. I discovered that if I use `"*"` as the event name, I can pass the event through. Is that expected? The visualizer declares this a "dynamic event" though I don't see any such thing in the docs.

**Reply by davidkpiano** [reply]
Yes, this is called a ["wildcard" event descriptor](https://xstate.js.org/docs/guides/transitions.html#wildcard-descriptors).
